src/admin-views/settings/tickets-commerce/paystack-promo.php

```diff
@@ -15,7 +15,7 @@ class="tec-tickets__admin-after-pay-promo-icon"
 			<?php esc_html_e( 'Add Paystack checkout to Tickets Commerce', 'event-tickets' ); ?>
 		</div>
 		<div class="tec-tickets__admin-after-pay-promo-description">
-			<?php echo wp_kses( esc_html_e( 'Accept payments for your ticket sales in Nigeria, Ghana, South Africa, and Kenya with Paystack.  Install the free plugin to get started.', 'event-tickets' ), 'post' ); ?>
+			<?php echo wp_kses( esc_html__( 'Accept payments for your ticket sales in Nigeria, Ghana, South Africa, and Kenya with Paystack.  Install the free plugin to get started.', 'event-tickets' ), 'post' ); ?>
```

Comment:
On a second look @DAnn2012, since we are escaping html, we don't need wp_kses at all here. It's just a waste. The output is going to be 100% safe. So we should do it like below:

Agreed ?

```suggestion
			<?php esc_html_e( 'Accept payments for your ticket sales in Nigeria, Ghana, South Africa, and Kenya with Paystack.  Install the free plugin to get started.', 'event-tickets' ); ?>
```